.github/workflows/ci.yml

```diff
@@ -10,10 +10,11 @@ jobs:
     uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.11.2
     with:
      run-unit-tests: true
-     run-integration-tests: true
+      # NOTE: Temporarily block integration-tests due to ibc version compatibility issue
+     run-integration-tests: false
```

Comment:
Why is this not working? I thought only e2e test is affected

Reply:
e2e and local integration tests share the same go.mod and it has conflicting version of ibc-go https://github.com/babylonlabs-io/babylon-sdk/blob/d68f9c24b971dca9d236adc2868e8a3eff6d3f8d/tests/e2e/go.mod#L41 and https://github.com/babylonlabs-io/babylon-sdk/blob/d68f9c24b971dca9d236adc2868e8a3eff6d3f8d/tests/e2e/go.mod#L116